config/environments/test.rb

```diff
@@ -29,10 +29,10 @@
     "Cache-Control" => "public, max-age=#{1.hour.to_i}"
   }
 
-  # Show full error reports and disable caching.
+  # Show full error reports and enable caching.
   config.consider_all_requests_local = true
-  config.action_controller.perform_caching = false
-  config.cache_store = :null_store
+  config.action_controller.perform_caching = true
+  config.cache_store = :memory_store
```

Comment:
Where does this get stored? Can you add notes

Reply:
Still trying to figure out caching myself, so the config is likely to change. Will add some notes. The point of this change is simply to allow tests to use cached values, which will speed them up massively in some cases (if I can figure out how to use caching properly!).